### DISCUSSION TITLE
Is it possible to authenticate using managed identity in the Azure pipeline?

### DISCUSSION THREAD

**Discussion by ztrhgf** [almenscorner/IntuneCD#166] [Q&A] [answered]
I am trying to figure out whether it is possible to use "[workload identity federation service connection](https://learn.microsoft.com/en-us/azure/devops/pipelines/library/connect-to-azure?view=azure-devops#create-a-new-workload-identity-federation-service-connection)" to authenticate instead of providing secret/certificate?

Seems like it is not supported?

**Comment by almenscorner** [comment, marked as answer]
Managed Identities is still not supported in Python MSAL library. I have implemented the supported options and are waiting for MS to add managed identity support.
